packages/harpparts/src/apparatus/utils/build-apparatus/index.ts

```diff
@@ -1,17 +1,38 @@
 import { reedArrayToMatrices } from '../reed-array-to-matrices'
 import type { Apparatus } from '../../types'
 import { ValvingIds } from '../../../valving'
-import type { TuningIds } from '../../../tuning'
+import { isReedArrayEmpty, TuningIds } from '../../../tuning'
 import { getTuning } from '../../../access-parts'
 
 export const buildApparatus = (
   tuningId: TuningIds,
   valvingId: ValvingIds
 ): Apparatus => {
-  const tuning = getTuning(tuningId)
+  const { reedArrays } = getTuning(tuningId)
+  const [reedArray1, reedArray2] = reedArrays
+  // TODO: the conditional on the possibility of reedArray2 being empty
```

Comment:
There's definitely some thought to put in here. This comment is enough for this code review. If I don't find a solution then a more detailed one will be due.

Reply:
The complexity here has been dealt with as part of #130.